Merged the ID back.

c-stephens opened the pull request:
I added the ID back in, so my fork will reflect it. But you can deny it, if you already did it.

I also forgot to add the correct author name when I upgraded my machine, so it automatically took me local username and hostname, so I changed those back too.

c-stephens committed:
Now compatiable with macOSs 10.15 Catalina.

c-stephens committed:
Added the bundle ID back.

c-stephens committed:
Updated README

c-stephens committed:
Fixed Typo in README

c-stephens committed:
Added link to new Repo

amoose136 committed:
Merge branch 'master' into master

amoose136 commented:
Will likely get one of the new Apple in-house silicon macs in the next year but until that happens this is an accurate statement.

c-stephens commented:
I just updated the workflow for Big Sur, and I added some features in like sending an iMessage from the workflow without having to open iMessages.

I also intend to get one of the new Apple Silicon Macs, but till then I cannot verify that this works on said hardware. 

If and when you want to nullify my statement, I will be more than glad to merge my current repo with yours. 

Also, it looks like someone "borrowed" a bit from our project. 

[Hermes](https://www.packal.org/workflow/hermes)
